problems/oldestPerson.js

```diff
@@ -11,8 +11,20 @@
  * ex: oldestPerson({ 'Marcus': 29, 'Julia': 29, 'Kevin': 34 })
  * returns: 'Kevin'
  */
-function oldestPerson() {
-
+function oldestPerson(obj) {
+    let out1 = "";
+    out = -Infinity;
+    for (let key in obj) {
+        if (key > out) {
+            key = out1;
+        }
+        if (key > out) {
+            out1 = key
+        }
```

Comment:
In this problem, the `key` variable inside of the `for in` is actually the names of each person. So we can use this to access the values of the keys like for example obj[key] would give is 29 for Marcus

Reply:
was getting the lowest age each instance tried.